src/server/api/routes/users/v2/nfts/featured.js

```diff
@@ -0,0 +1,77 @@
+const { Router } = require('express');
+const {
+  authenticateV2Login,
+  checkWalletMatch,
+} = require('../../../../middleware/auth');
+const { setPrivateCacheHeader } = require('../../../../middleware/cache');
+const { handleRestfulError } = require('../../../../middleware/error');
+const { isValidAddress } = require('../../../../util/cosmos');
+const {
+  FieldValue,
+  walletUserCollection,
+} = require('../../../../../modules/firebase');
+
+const router = Router();
+
+router.get('/v2/users/:wallet/nfts/featured', async (req, res, next) => {
+  try {
+    const { wallet: user } = req.params;
+    if (!isValidAddress(user)) {
+      res.status(400).send('INVALID_ADDRESS');
+      return;
+    }
+    const userDoc = await walletUserCollection(user).get();
+    const { featuredNFTClassIds = [] } = userDoc.data();
+    res.json({ featured: featuredNFTClassIds });
+  } catch (err) {
+    handleRestfulError(req, res, next, err);
+  }
+});
+
+router.post(
+  '/v2/users/:wallet/nfts/featured',
+  authenticateV2Login,
+  checkWalletMatch,
+  async (req, res, next) => {
+    try {
+      setPrivateCacheHeader(res);
+      const { wallet: user } = req.params;
+      const { nftClassIds = [] } = req.query;
+      if (!nftClassIds.length) {
+        res.status(400).send('NFT_CLASS_ID_MISSING');
+        return;
+      }
```

Comment:
I think we prefer update payload as body in this case since clickjack

Reply:
I see, also wondering whether the same issue should be a concern on `DELETE` endpoints?

Reply:
Or there isn't motivation for a hacker to `DELETE` your featured/hidden classes in our scenario?

Reply:
I dont think it is possible to make a form submission send a `DELETE`, but need to verify